interfaces/cython/cantera/solutionbase.pyx

```diff
@@ -520,6 +519,13 @@ cdef class SolutionArrayBase:
         dest.base = dest._base.get()
         return dest
 
+    def __repr__(self):
+        try:
+            width = get_terminal_size().columns
+        except:
+            width = 100
+        return self.info(width=width)
```

Comment:
Should the default width here be 80, corresponding to the default of the `info` function (or vice versa)?

Reply:
This is deliberate and application specific: for `__repr__`, the column width is detected for a terminal, while it cannot be detected for a jupyter notebook (where 80 would be too narrow). For `info`, 80 is simply a conservative estimate for width presets. I can obviously make `info` smarter in that sense?

Reply:
I made detection of text width automatic for `info` as well, so both methods now make the same assumptions. That way, we can use a less conservative estimate than 80. On that note, it is possible to use html formatting for Jupyter, but I'd like to leave this for another time (or another contributor).